tornado_swagger/setup.py

```diff
@@ -39,6 +40,26 @@ def export_swagger(
     )
 
 
+def swagger_decorator(
+        *,
+        responses: typing.Dict[int, typing.Dict[str, typing.Any]],
+        request: typing.Optional[typing.Type[BaseModel]] = None,
+        query: typing.Optional[typing.Type[BaseModel]] = None,
+        tags: typing.Optional[typing.List[str]] = None
+):
+    def decorator(f: typing.Callable) -> typing.Callable:
+        setattr(f, "_is_swagger_handler", True)
+        setattr(f, "_response_models", responses)
+        if request:
+            setattr(f, "_request_model", request)
+        if query:
+            setattr(f, "_query_params", query)
+        if tags:
+            setattr(f, "_swagger_tags", tags)
+        return f
+    return decorator
```

Comment:
IMO, its better to use a single additional attribute like '_swagger_info':
* decrease chances for possible conflicts
* you can use a structure for it's value like: `class SwaggerMethodInfo` with a fixed set of fields